<launch>
  <node name="video_pub" pkg="simple_webcam" type="video_pub.py" />

  <node name="image_proc_server" pkg="simple_webcam" type="image_proc_server.py" />

  <node name="image_proc_client" pkg="simple_webcam" type="image_proc_client.py" />

</launch>